### PR TITLE
Adding template page

### PR DESCRIPTION
**Pull request by monkeyninja007**
moving code from index to _layout

### CODE REVIEW

File: Service_De_Garde/Service_De_Garde/Pages/Index.cshtml [hunk 1]
```diff
@@ -280,63 +262,14 @@
...
-          <h2 class="text-white m-0">Bring Fun Life To Your Kids</h2>
-          <a href="#" class="btn btn-primary btn-custom-1 py-3 px-5 ml-auto">Get Started</a>
-          </div>
-        </div>
-      </div>
-    </div>
-
     </div>
 
     <script src="js/jquery-3.3.1.min.js"></script>
```
Comment (shnitze) [inline Service_De_Garde/Service_De_Garde/Pages/Index.cshtml]: Script links should go in ValidationScriptPartial

Also, there is an html closing tag that does not belong...

File: Service_De_Garde/Service_De_Garde/Pages/Shared/_Layout.cshtml [hunk 2]
```diff
@@ -1,16 +1,93 @@
...
 
+        <footer class="site-footer">
+            <div class="container">
+                <div class="row">
+                    <div class="col-lg-4">
+                        <h2 class="footer-heading mb-3">About Us</h2>
+                        <p class="mb-5">Far far away, behind the word mountains, far from the countries Vokalia and Consonantia, there live the blind texts. </p>
+
+                        <h2 class="footer-heading mb-4">Newsletter</h2>
+                        <form action="#" class="d-flex" class="subscribe">
```
Comment (shnitze) [inline Service_De_Garde/Service_De_Garde/Pages/Shared/_Layout.cshtml]: There's 2 class attributes within 1 html tag..
```suggestion
                        <form action="#" class="d-flex subscribe">
```

File: Service_De_Garde/Service_De_Garde/Pages/Shared/_Layout.cshtml [hunk 3]
```diff
@@ -1,16 +1,93 @@
...
 
-    <script src="~/lib/jquery/dist/jquery.min.js"></script>
-    <script src="~/lib/bootstrap/dist/js/bootstrap.bundle.min.js"></script>
-    <script src="~/js/site.js" asp-append-version="true"></script>
+                </div>
+            </div>
+        </footer>
 
-    @RenderSection("Scripts", required: false)
+        <script src="~/lib/jquery/dist/jquery.min.js"></script>
```
Comment (shnitze) [inline Service_De_Garde/Service_De_Garde/Pages/Shared/_Layout.cshtml]: The links should also go in ValidationScriptPartial

File: Service_De_Garde/Service_De_Garde/Pages/Index.cshtml (hunk 1 above)
Comment (monkeyninja007) [reply Service_De_Garde/Service_De_Garde/Pages/Index.cshtml]: Changes were made

File: Service_De_Garde/Service_De_Garde/Pages/Shared/_Layout.cshtml (hunk 2 above)
Comment (monkeyninja007) [reply Service_De_Garde/Service_De_Garde/Pages/Shared/_Layout.cshtml]: html tag is now <form action=# class="d-flex subscribe">

**Pull request merged**